app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -3,61 +3,66 @@
 @import 'dropzone';
 
 .tabbable-panes {
-    min-height: 500px;
-
-    .nav-pills {
-        background-color: var(--stanford-30-black);
-        border-radius: var(--bs-nav-pills-border-radius);
-        
-        .nav-link:not(.active) {
-            color: var(--stanford-black);
-            border-bottom: 1px solid var(--stanford-10-black);
-            border-radius: unset;
-        }
-        
-        .nav-link.active {
-            background-color: var(--stanford-black);
-            color: var(--stanford-10-black);
-            font-weight: bold;
-            
-            &::before {
-                font-family: 'bootstrap-icons';
-                display: inline-block;
-                vertical-align: -10%;
-                margin-right: 4px;
-                content: "\F138"; // right arrow
-            }
-        }
-
-        .nav-link.is-invalid {
-            color: var(--bs-danger-text-emphasis);
-
-            &::after {
-                font-family: 'bootstrap-icons';
-                display: inline-block;
-                vertical-align: -10%;
-                margin-left: 10px;
-                content: "\F33A"; // exclamation triangle fill
-            }
-        }
+  min-height: 500px;
+
+  .nav-pills {
+    background-color: var(--stanford-10-black);
+    border-radius: var(--bs-nav-pills-border-radius);
+
+    .nav-link:not(.active) {
+      color: var(--stanford-black);
+      border-bottom: 1px solid var(--stanford-10-black);
+      border-radius: unset;
     }
 
-    // Legends by default have a larger font. When used as a form label, want a smaller font.
-    legend.form-label {
-        font-size: unset;
+    .nav-link.active {
+      background-color: var(--stanford-black);
+      color: var(--stanford-10-black);
+      font-weight: bold;
+
+      &::before {
+        font-family: 'bootstrap-icons';
+        display: inline-block;
+        vertical-align: -10%;
+        margin-right: 6px;
+        content: "\F138"; // right arrow
+      }
     }
+
+    .nav-link.is-invalid {
+      color: var(--bs-danger-text-emphasis);
+
+      &.active {
+        background-color: white;
+        border: 3px solid var(--stanford-black)
+      }
```

Comment:
Style the active invalid tab with a white bg color and a black border per #186 mock-ups.